src/components/WatchlistPopup/WatchlistGQL.js

```diff
@@ -1,6 +1,7 @@
 import gql from 'graphql-tag'
+import { generalData, project } from '../../pages/Projects/allProjectsGQL'
 
-export const WatchlistGQL = gql`
+export const allWatchlistsGQL = gql`
```

Comment:
The updated scheme for naming queries/mutation is following: `ALL_WATCHLISTS_QUERY` (`_MUTATION` for mutations)